src/deepsparse/operators/operator.py

```diff
@@ -137,3 +138,18 @@ def yaml(self):
 
     def json(self):
         pass
+
+
+class Operator(BaseOperator):
```

Comment:
let's merge this into the main operator class

Reply:
ok